config/mcu_definitions/toolhead/BTT_SB2209_v1.0.cfg

```diff
@@ -0,0 +1,22 @@
+[board_pins toolhead_manufacturer]
+mcu: toolhead
+aliases:
+   MCU_TMCDRIVER_STEP=PD0 , MCU_TMCDRIVER_DIR=PD1 , MCU_TMCDRIVER_ENABLE=PD2 , MCU_TMCDRIVER_UART=PA15 ,
+   MCU_TMCDRIVER_DIAG=PB3 ,
```

Comment:
As per the documentation (see here: https://cdn.shopify.com/s/files/1/1619/4791/t/48/assets/-8-1676011507868.jpg?v=1676011516)

I can see that motor pins are named "MOTOR Drive" and with some slight change also to the different pins names. So I would suggest using something like: `MCU_MOTORDRIVE_STEP`, `MCU_MOTORDRIVE_DIR`, `MCU_MOTORDRIVE_EN`, `MCU_MOTORDRIVE_UART` and `MCU_MOTORDRIVE_DIAG`

Reply:
For archive sake, i highly suggest using original sources of the document and not links to a webshop CDN.